woocommerce/woocommerce
# Split the Extensions page in WP Admin and add respective menu items

### All Submissions:






### Changes proposed in this Pull Request:
- Register a new top-level menu for Marketplace and create two subpages for Extensions and My Subscriptions
- register new page IDs for loading WooCommerce scripts and styles
- Add a method in `WC_Admin_Addons` for Helper page output only
- remove top nav from the Extensions page


### How to test the changes in this Pull Request:

1. Check out this branch on local environment.
2. Go to WP Admin and observe there's a new top-level menu called Marketplace with two subpages called Extensions and My Subscriptions.
3. Open each of these pages and make sure they load correctly and have all styles in place.
4. Make sure the correct menu item is highlighted when you open each page.

### Other information:


### Screenshots

![](https://d.pr/i/fYtxg2+)

![](https://d.pr/i/jbTHWB+)

### Notes
- didn't remove top nav on the My Subscriptions page yet because editing this file triggered a lot of phpcs errors. I'll take care of it later.
